edgelet/aziot-edged/src/main.rs

```diff
@@ -70,6 +72,10 @@ async fn run() -> Result<(), EdgedError> {
         )
     })?;
 
+    let migc_filename = settings.homedir().join("migc");
+    log::info!("creating file at: {}", migc_filename.to_str().unwrap());
```

Comment:
delete this

Reply:
We should actually keep it around until we validate why isn't working in package scenario.